op-challenger/game/fault/trace/vm/kona_vm_config.go

```diff
@@ -0,0 +1,51 @@
+package vm
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/trace/utils"
+	"github.com/ethereum-optimism/optimism/op-node/chaincfg"
+)
+
+type KonaVmConfig struct {
+	Config
+}
+
+var _ VmConfig = (*KonaVmConfig)(nil)
+
+func NewKonaVmConfig(vmConfig Config) *KonaVmConfig {
+	return &KonaVmConfig{
+		vmConfig,
+	}
+}
+
+func (s *KonaVmConfig) Cfg() Config {
+	return s.Config
+}
+
+func (s *KonaVmConfig) FillHostCommand(args []string, dataDir string, inputs utils.LocalGameInputs) ([]string, error) {
+	if args == nil {
+		return nil, errors.New("args is nil")
+	}
+	if s.Network == "" {
+		return nil, errors.New("Network is not defined")
+	}
+
+	chainCfg := chaincfg.ChainByName(s.Network)
+	args = append(args,
+		"--",
```

Comment:
Can I suggest that this `--` is added by the cannon executor rather than the `VmConfig` - it's not part of the CLI command for kona, but actually part of cannon's CLI syntax to tell it when the server command starts.